build.sbt

```diff
@@ -152,10 +152,34 @@ lazy val amm = project
 
 
     assemblyOption in assembly := (assemblyOption in assembly).value.copy(
-      prependShellScript = Some(
+      prependShellScript = {
+        def universalScript(shellCommands: Seq[String],
+                            cmdCommands: Seq[String],
+                            shebang: Boolean = true): Seq[String] = {
+          Seq(
+            Seq("#!/usr/bin/env sh")
+              .filter(_ => shebang),
+            Seq("shopt -s expand_aliases", "alias ::=''")
+              .map(line => s":; $line"),
+            (shellCommands :+ "exit")
+              .map(line => s":: $line"),
+            "@echo off" +: cmdCommands :+ "exit /B",
+            Seq("\r\n")
+          ).flatten
+        }
+
+        def defaultUniversalScript(javaOpts: Seq[String] = Seq.empty, shebang: Boolean = true): Seq[String] = {
```

Comment:
`shebang` defaults to `true` gives me:

```
'#!' is not recognized as an internal or external command,
operable program or batch file.
```

Defaulting to `false` works for both sh and batch modes (Windows 10).
Any reason why it defaults to `true`?

Reply:
Yes, this is an unfortunate tradeoff. If the shebang line is disabled you cannot simply run "./amm" on linux.

Reply:
Hmm, perhaps a different shell might not like it; I tried without it on macOS's terminal (`GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin17)`) and KDE Neon's Konsole (`GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)`). 
At any rate, I think it's a better trade-off to enhance the curl+chmod command in Ammonite installation guide to prepend the shebang line (rather than all batch users getting the above error message).